Register and Edit sample batches with XLSX sheets (#30)
qbicsoftware/research-data-management: docs/batch/sample-batch.md

</br>
Please go ahead and **upload your filled metadata file** in the dialog.
Once the data manager validated the information in your file,
go ahead and **choose a name for your batch** if you want to change it.
</br>
Once you named your batch and uploaded the file with the necessary information, go ahead and **click the
`Edit batch`** button. The data manager will now go through the process of updating the sample information within your
experiment.

After the data manager is done registering your samples, you can **close** the dialog **by clicking the
`Finish`** button.

[//]: # (How do I delete existing samples in my experiment?)
## Delete a sample batch

!!! warning "Batch Deletion"
    Keep in mind, that deleting a batch will also delete all sample metadata of the samples within the batch

To **start** with the sample batch deletion process, click on the **`delete`** button next to the batch in question within the action column.

!!! info "Project role"
    Should you not see the action column,   
    please make sure that you have been granted the "write" or "admin" role to it by the project owner/admin.

> This will open the batch deletion dialog requiring **validation** of the batch deletion process by clicking the **`Confirm`** button.

Author: ```suggestion
This will open the batch deletion dialog requiring **confirmation** of the batch deletion process by clicking the **`Confirm`** button.
```

!!! warning "Attached Measurements"
    Keep in mind, that batches can only be deleted if none of the samples within the batch have been used in a measurement.)
    Otherwise, you need to delete the measurements in question before the batch can be deleted.

[//]: # (How do I download sample metadata)
## Download sample metadata

To **download** the sample metadata for all registered samples click on the **`Download sample metadata`** button.

> This will export the provided registered metadata as an `.xlsx` file to your local download directory.

Author: ```suggestion
This will export all registered metadata as an `.xlsx` file to your local download directory.
```
